citadel/indico_citadel/util_test.py

```diff
@@ -18,7 +18,7 @@
     ('hey title:something', '+title:something +(hey)'),
     ('title:something hey', '+title:something +(hey)'),
     ('hey title:something hey person:john', '+title:something +person:john +(hey hey)'),
-    ('<*\\^()', '+(\\<\\*\\\\\\^\\(\\))'),
+    ('<*\\^()', '+(\\<*\\\\\\^\\(\\))'),
```

Comment:
Maybe we could add another more "realistic" case here? Like searching for `foo*` to make it clear it not being escaped is intended and not a mistake?